server/protocols/jmap-draft/pom.xml

```diff
@@ -27,10 +27,10 @@
         <relativePath>../../pom.xml</relativePath>
```

Comment:
Please add a `.draft.` part in the package name in this module too.

Reply:
Done.